Sources/ImperialShopify/Shopify.swift

```diff
@@ -2,30 +2,27 @@
 import Vapor
 
 public final class Shopify: FederatedService {
+    public var tokens: any FederatedServiceTokens { self.router.tokens }
+    public var router: any FederatedServiceRouter { self.shopifyRouter }
 
-    public var tokens: FederatedServiceTokens { self.router.tokens }
-    public var router: FederatedServiceRouter { self.shopifyRouter }
-
-    public var shopifyRouter: ShopifyRouter
+    public let shopifyRouter: ShopifyRouter
```

Comment:
Why use `var` instead of `let` since we don’t need computed properties?

Reply:
Do we need `shopifyRouter` here?

Reply:
No we don't